docs/adding-kubernetes-versions.md

```diff
@@ -38,6 +38,10 @@ Once new Docker images are ready, KKP can be updated as well.
   minor (just copy an existing job and adjust accordingly). Make sure to change the Docker
   image tag for the e2e images to use the new tag you just created with the new test binaries.
 - Update the CSI addon manifests (`addon/csi/*.yaml`) to include the new minor version.
+- Update the OpenStack CCM manifest (`pkg/resources/cloudcontroller/openstack.go`) to
+  include the new minor version.
+  - The latest OpenStack CCM version can be found in the 
+  [`kubernetes/cloud-provider-openstack` repository](https://github.com/kubernetes/cloud-provider-openstack).
```

Comment:
Aren't this change from another PR (https://github.com/kubermatic/kubermatic/pull/6300) and two PRs got mixed?

Reply:
How TF did that happen?!

Reply:
Oooooh I get it. My bad.

Reply:
Fixed.

In my previous PR for the 2.14 branch, I fixed a `git commit` by running `git commit --amend`. And when I made the same changes for 2.15, I just pressed up in my shell and accidentally amended Marko's last commit.